src/Features/Core/Portable/RemoveUnnecessaryParentheses/AbstractRemoveUnnecessaryParenthesesDiagnosticAnalyzer.cs

```diff
@@ -21,12 +22,14 @@ internal abstract class AbstractRemoveUnnecessaryParenthesesDiagnosticAnalyzer<
 
         /// <summary>
         /// A diagnostic descriptor that will fade the span (but not put a message or squiggle).
+        /// Not configurable so that the severity cannot be raised and end up in the error list.
         /// </summary>
         private static readonly DiagnosticDescriptor s_diagnosticWithFade = CreateDescriptorWithId(
                 IDEDiagnosticIds.RemoveUnnecessaryParenthesesDiagnosticId,
                 new LocalizableResourceString(nameof(FeaturesResources.Remove_unnecessary_parentheses), FeaturesResources.ResourceManager, typeof(FeaturesResources)),
                 string.Empty,
-                isUnneccessary: true);
+                isUnneccessary: true,
+                customTags: WellKnownDiagnosticTags.NotConfigurable);
```

Comment:
Will this have any effect, given we also have another non-fading descriptor below with same ID but that one is configurable?

Reply:
No I don't believe so, the main one is still configurable.  I'm still able to modify the severity with editor config and rulesets, and one of the unit tests checks that when it is changed to error, it gets reported as error.

Reply:
Should we instead just use a different diagnostic ID here, similar to [here](http://source.roslyn.io/#Microsoft.CodeAnalysis.Features/RemoveUnnecessaryImports/AbstractRemoveUnnecessaryImportsDiagnosticAnalyzer.cs,21). I think that should ensure that you won't need any further changes in this file, other then marking the fading descriptor as NotConfigurable.